docs/learn/tutorials/versioned/samza-sql.md

```diff
@@ -0,0 +1,129 @@
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+
+There are couple of ways to use Samza SQL
+
+1. Run Samza SQL on your local machine.
+2. Run Samza SQL on YARN.
+
+# Running Samza SQL on your local machine  
+
+Samza SQL console tool documented [here](https://github.com/srinipunuru/samza-sql-tools/blob/master/README.md) uses Samza standalone to run the Samza SQL on your local machine. This is the quickest way to play with Samza SQL. Please follow the instructions [here](https://github.com/srinipunuru/samza-sql-tools/blob/master/README.md) on how to use the Samza SQL console to run stream processing jobs on your local machine using SQL.
+
+# Running Samza SQL on YARN
+
+The [hello-samza](https://github.com/apache/samza-hello-samza) project is an example project designed to help you run your first Samza application. It has examples of applications using the low level task API, high level API as well as Samza SQL.
+
+This tutorial demonstrates a simple Samza application that uses SQL to perform stream processing.
+
+### Get the Code
+
+Check out the hello-samza project:
+
+{% highlight bash %}
+git clone https://git.apache.org/samza-hello-samza.git hello-samza
+cd hello-samza
+git checkout latest
+{% endhighlight %}
+
+This project contains everything you'll need to run your first Samza application.
+
+### Start a Grid
+
+A Samza grid usually comprises three different systems: [YARN](http://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/YARN.html), [Kafka](http://kafka.apache.org/), and [ZooKeeper](http://zookeeper.apache.org/). The hello-samza project comes with a script called "grid" to help you setup these systems. Start by running:
+
+{% highlight bash %}
+./bin/grid bootstrap
+{% endhighlight %}
+
+This command will download, install, and start ZooKeeper, Kafka, and YARN. It will also check out the latest version of Samza and build it. All package files will be put in a sub-directory called "deploy" inside hello-samza's root folder.
+
+If you get a complaint that JAVA_HOME is not set, then you'll need to set it to the path where Java is installed on your system.
+
+Once the grid command completes, you can verify that YARN is up and running by going to [http://localhost:8088](http://localhost:8088). This is the YARN UI.
+
+### Create the topic and Generate kafka events
+
+The sample requires a topic named PageViewStream to be created on the kafka broker. You can follow the instructions in the (kafka quick start guide)[http://kafka.apache.org/quickstart] to create a topic named "PageViewStream". 
+
+Once you created the topic, You need to generate few events into the topic so that the SQL can process those events.
+
+Please follow the instructions in (Samza SQL tools)[https://github.com/srinipunuru/samza-sql-tools/blob/master/README.md] to generate events of type ProfileChangeEvent into the topic named ProfileChangeStream.
+
+### Build a Samza Application Package
+
```

Comment:
This is also dups of hello-samza doc, right? I think that we can refer to the single doc instead of repeating the same content here.